File: lista1/ex12.py
rm = int(input("Informe o RM: "))
soma = 0

resto = rm % 10
soma = soma + resto
rm = rm // 10

resto = rm % 10
soma = soma + resto
rm = rm // 10

resto = rm % 10
soma = soma + resto
rm = rm // 10

resto = rm % 10
soma = soma + resto
rm = rm // 10

resto = rm % 10
soma = soma + resto
rm = rm // 10

print(soma)

File: lista1/ex12.c
#include<stdio.h>
int main() {int rm;printf("Digite o rm: ");scanf("%d", &rm);int soma = 0;int resto = rm % 10;
    soma = soma + resto;
    rm = rm / 10;

    resto = rm % 10;
    soma = soma + resto;
    rm = rm / 10;

    resto = rm % 10;
    soma = soma + resto;
    rm = rm / 10;

    printf("A soma vale %d", soma)
}